<launch>

  <node pkg="loam_interface" exec="loamInterface" name="loamInterface" output="screen">
    <!-- <param name="stateEstimationTopic" value="/integrated_to_init" />
    <param name="registeredScanTopic" value="/velodyne_cloud_registered" /> -->
    <param name="stateEstimationTopic" value="/lio_odom" />
    <param name="registeredScanTopic" value="/real_points_raw" />
    <param name="flipStateEstimation" value="false" />
    <param name="flipRegisteredScan" value="false" />
    <param name="combinedOdom" value="true" />
    <param name="sendTF" value="false" />
    <param name="reverseTF" value="false" />
    <param name="odom_frame" value="ig_odom" />
  </node>

</launch>
